io_scene_godot/converters/animation/constraint_baking.py

```diff
@@ -74,7 +74,7 @@ def bake_constraint_to_action(blender_object, base_action, bake_type,
         )
 
         bpy.context.scene.objects.active = active_obj_backup
-    else:
+    elif bpy.app.version <= (2, 80, 0):
```

Comment:
If you're saying that based on my comment from above:

> I'm unclear on why 2.79.6 seems to have the 2.80 API

That was a misunderstanding on my part, based on how the arch package is built.

Reply:
I am only curious about why `>= (2, 80, 0)` instead of `> (2, 80, 0)`.  If Blender retain the old API in version 2.80.0, then it is strange🤔.     If you are sure about what you are doing is right, then it is fine

Reply:
aha! yes, `<=` is wrong, will fix that tonight.